packages/babel-plugin-transform-es2015-modules-commonjs/README.md

```diff
@@ -62,6 +62,69 @@ require("babel-core").transform("code", {
 });
 ```
 
+## Options `spec`
+
+By default, `babel` actually implements importing very loosely, by
+supporting treating a commonjs export as if it was a namespace export.
+The exported namespace is also not frozen and has an incorrect prototype.
+
+The `spec` option, when set to `true`, tries to generate code that is as
+close as possible to what is required by the ECMA262 spec without relying
+on `Proxy`. The exports will be frozen, and imports will always be treated
+like ES modules.
+
+Importing a commonjs module (say, the standard `fs` module) will always
+wrap it in an ES module that has a single `default` export. This means that
+some imports that work in non-`spec` mode, like `import { readFile } from 'fs'`,
```

Comment:
Possible. Spec does not explicitly ban it from happening. Implementations have concerns and order of operations may be hard for codegen. Still in talks.